yt_dlp/compat.py

```diff
@@ -133,6 +133,16 @@ def compat_asyncio_run(coro):
     asyncio.run = compat_asyncio_run
 
 
+try:  # <= 3.6
+    compat_asyncio_all_tasks = asyncio.tasks.Task.all_tasks
+except AttributeError:  # > 3.7
+    compat_asyncio_all_tasks = asyncio.tasks.all_tasks
```

Comment:
```suggestion
try:  # >= 3.7
    asyncio.tasks.all_tasks
except AttributeError:
    asyncio.tasks.all_tasks = asyncio.tasks.Task.all_tasks
```
I was thinking of doing it this way. On one hand, we are mutating a standard library. But we are also only adding a new function, and this makes removing the compat code way easier in the future. Any downsides to this?

Reply:
maybe that's okay